.github/workflows/build-test.yaml

```diff
@@ -2014,13 +2220,15 @@ jobs:
         with:
           test-focus: 'Min KOTS Version'
           kots-namespace: 'min-kots-version'
-          k8s-version: '${{ matrix.k8s_version }}'
-          k8s-arguments: ${{ startsWith(matrix.k8s_version, 'v1.22') && '--egress-selector-mode disabled' || '' }}
+          k8s-distribution: ${{ matrix.cluster.distribution }}
+          k8s-version: ${{ matrix.cluster.version }}
           testim-access-token: '${{ secrets.TESTIM_ACCESS_TOKEN }}'
           testim-branch: ${{ github.head_ref == 'main' && 'master' || github.head_ref }}
           kotsadm-image-registry: ttl.sh
           aws-access-key-id: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_ACCESS_KEY_ID }}'
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
+          replicated-api-token: '${{ secrets.C11Y_MATRIX_TOKEN }}'
+          k8s-cluster-skip-teardown: true
```

Comment:
This one and the "target kots version" test was a little tricky.  We run the test using the `e2e` action, but also need the cluster to be around for the CLI step that comes after.  May need to tweak the `e2e` action to export the cluster ID so we can delete it?